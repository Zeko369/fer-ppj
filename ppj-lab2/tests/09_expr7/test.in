IDN 1 a
OP_PRIDRUZI 1 =
BROJ 1 1
OP_PLUS 1 +
L_ZAGRADA 1 (
L_ZAGRADA 1 (
BROJ 1 25
OP_MINUS 1 -
BROJ 1 32
D_ZAGRADA 1 )
OP_DIJELI 1 /
BROJ 1 5
OP_PLUS 1 +
BROJ 1 12
OP_MINUS 1 -
BROJ 1 11
D_ZAGRADA 1 )
OP_PUTA 1 *
L_ZAGRADA 1 (
BROJ 1 1
OP_MINUS 1 -
BROJ 1 2
D_ZAGRADA 1 )
