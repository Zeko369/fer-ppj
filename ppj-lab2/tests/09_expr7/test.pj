a = 1+((25-32)/5 + 12-11) * (1-2)
