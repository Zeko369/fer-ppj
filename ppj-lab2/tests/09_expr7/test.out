<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 a
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 1
      <T_lista>
       $
     <E_lista>
      OP_PLUS 1 +
      <E>
       <T>
        <P>
         L_ZAGRADA 1 (
         <E>
          <T>
           <P>
            L_ZAGRADA 1 (
            <E>
             <T>
              <P>
               BROJ 1 25
              <T_lista>
               $
             <E_lista>
              OP_MINUS 1 -
              <E>
               <T>
                <P>
                 BROJ 1 32
                <T_lista>
                 $
               <E_lista>
                $
            D_ZAGRADA 1 )
           <T_lista>
            OP_DIJELI 1 /
            <T>
             <P>
              BROJ 1 5
             <T_lista>
              $
          <E_lista>
           OP_PLUS 1 +
           <E>
            <T>
             <P>
              BROJ 1 12
             <T_lista>
              $
            <E_lista>
             OP_MINUS 1 -
             <E>
              <T>
               <P>
                BROJ 1 11
               <T_lista>
                $
              <E_lista>
               $
         D_ZAGRADA 1 )
        <T_lista>
         OP_PUTA 1 *
         <T>
          <P>
           L_ZAGRADA 1 (
           <E>
            <T>
             <P>
              BROJ 1 1
             <T_lista>
              $
            <E_lista>
             OP_MINUS 1 -
             <E>
              <T>
               <P>
                BROJ 1 2
               <T_lista>
                $
              <E_lista>
               $
           D_ZAGRADA 1 )
          <T_lista>
           $
       <E_lista>
        $
  <lista_naredbi>
   $
